crates/rpc-client/src/call.rs

```diff
@@ -81,11 +80,17 @@ where
                 self.set(CallState::Complete);
                 return Ready(RpcResult::Err(e));
             }
-            let request = request.take().expect("no request").serialize();
 
+            let request = request.take().expect("no request");
+            debug!(method=%request.meta.method, id=%request.meta.id, "sending request");
```

Comment:
should we log the entire request object?

Reply:
Yes I think so, added a trace! with ?request and params type name